src/main/java/graphql/schema/GraphQLArgument.java

```diff
@@ -71,10 +73,15 @@ private GraphQLArgument(String name, String description, GraphQLInputType type,
     }
 
 
+    @Deprecated
```

Comment:
why is this deprecated now?

Reply:
@andimarek This method was part of older recursive reference resolution algorithm.
`getChildren()` unwraps actual type and `replaceType(GraphQLInputType type)` performs simple type set.

I am not sure if anyone else is using it, so I have marked it as @deprecated.
Ideally, if it is not part of some use case, this can be removed in future versions.

If I miss some use case, I would be happy to learn about it and add support.

Reply:
please remove them: they are not used anywhere else and were only for one purpose